app/src/main/java/team7/voluntime/Activities/EventRegisterActivity.java

```diff
@@ -53,6 +74,21 @@ protected void onCreate(Bundle savedInstanceState) {
         Event event = intent.getParcelableExtra("event");
 
         eventID = event.getId();
+        charityID = event.getOrganisers();
+        nReference = database.getReference("Charities").child(charityID);
```

Comment:
if nReference is referring to the particular charity, could we rename it to like `charityReference` or something?